components/page-footer/src/_mixin.scss

```diff
@@ -12,13 +12,13 @@
 }
 
 @mixin utrecht-page-footer {
-  --utrecht-document-color: currentColor;
-  --utrecht-heading-color: currentColor;
-  --utrecht-link-color: currentColor;
-  --utrecht-link-focus-color: currentColor;
-  --utrecht-link-hover-color: currentColor;
-  --utrecht-link-active-color: currentColor;
-  --utrecht-link-visited-color: currentColor;
+  --utrecht-document-color: var(--utrecht-footer-document-color, currentColor);
+  --utrecht-heading-color: var(--utrecht-footer-heading-color, currentColor);
+  --utrecht-link-color: var(--utrecht-footer-link-color, currentColor);
+  --utrecht-link-focus-color: var(--utrecht-footer-link-focus-color, currentColor);
+  --utrecht-link-hover-color: var(--utrecht-footer-link-hover-color, currentColor);
+  --utrecht-link-active-color: var(--utrecht-footer-link-active-color, currentColor);
+  --utrecht-link-visited-color: var(--utrecht-footer-link-visited-color, currentColor);
```

Comment:
@Marwaxhello Ik denk dat het misschien ook werkt (en dan simpeler is) om de oude code te verplaatsen naar `utrecht-page-footer__content`

Het gedrag zal een paar weken geleden kapot zijn gegaan door de introductie van `utrecht-page-footer__content`.

Wat dat betreft zou dit een `patch` zijn en niet `minor`